wizard/src/utils/detect_place.ts

```diff
@@ -13,9 +13,67 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { ConfidenceLevel, DetectedDetails, DetectedFormat } from "../types";
+import {
+  ConfidenceLevel,
+  DetectedDetails,
+  PlaceProperty,
+  PlaceType,
+  TypeProperty,
+} from "../types";
 import countriesJSON from "./country_mappings.json";
 
+const MIN_HIGH_CONF_DETECT = 0.9;
+
+// All supported Place types must be encoded below.
+const PLACE_TYPES = new Map<string, PlaceType>([
```

Comment:
Document what the key is?  Also I don't follow why the key is different from typeName, though they seem to match 1:1.

Reply:
Ditto with PLACE_PROPERTIES below

Reply:
Removed the key from here (not using the map structure here) and instead creating in-memory maps during initialization using the dcNames as keys.